config/routes.rb

```diff
@@ -3,6 +3,8 @@
 Rails.application.routes.draw do
   devise_for :users
   root "home#index"
-  resources :ingredients
-  resources :users, only: %i[index edit update destroy]
+  resources :appliances, except: %i[show]
+  resources :ingredients, except: %i[show]
+  resources :ingredient_categories, except: %i[show]
```

Comment:
No needed route 👀

Reply:
Which one is not needed? The users one? :o

Reply:
Do you have a `ingredient_categories` model? 👁️ 
I don't think so
<img width="377" alt="Screen Shot 2022-01-28 at 10 23 51" src="https://user-images.githubusercontent.com/26292224/151583736-b1137f51-216d-42fc-b8aa-b18121d88345.png">